src/calculateRentalCost.js

```diff
@@ -4,7 +4,30 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
```

Comment:
You can eliminate the `sum` variable by returning the calculated value directly in each condition block. This makes the code cleaner and more concise.